model/godleyIcon.cc

```diff
@@ -38,7 +38,7 @@ namespace minsky
     struct OrderByName
     {
       bool operator()(const VariablePtr& x, const VariablePtr& y) const
-      {assert(x&&y); return x->valueId() < y->valueId();}
+      {assert(x&&y); return x->name() < y->name();}    // Characeters stripped from valueIds by stripActive() seem to cause this to not work properly. For ticket 1165.
```

Comment:
This will fix the bug you're targetting, but will create another one. In a global godley table, variables "foo" and ":foo" are in fact the same variable. In godley tables stashed in a group, "foo" and ":foo" will be different variables.

The problem is clearly due to distinctly different variables having the same valueId, which is wjat ticket #729 is about.

Reply:
Oy vey, that's true. OK, putting a latexToPango around the valueId seems to work too. So, I've done that. Just running "make sure" on my machine now to check that all tests pass and will commit the code soon after. I will also try the approach of putting latexToPango around valueIds when they are created and hopefully this will get around the failing tests.

Reply:
Errm, I was wrong... I made the changes but failed to compile before running Minsky to test it. After compiling via make sure I decided to double check and the bug was back... I really wanted to kill this one today, but I'm out of steam after battling with the installation of curl too. (I've started working on a TCL implementation that will import CSVs from URLs, because in the best case scenario, users will have to install curl's libraries on their system if we wanted to go that way. I could not get it to work as a submodule and had to install the dev package separately to get access to the curl.h header).

Reply:
Readjusting the saved logs to account for changes to the value id is not "Volkswagening". That should be the only impact on the tests IIUC.

Reply:
Done. The tests all pass now and hopefully they will on Travis too.